### DISCUSSION TITLE
A complete guide to self-hosting supabase

### DISCUSSION THREAD

**Discussion by globemediaofficial** [supabase/supabase#11254] [General]
I struggled with self-hosting in the beginning because there was so much left to figure out on my own with the minimal documentation that Supabase provides. Now I see people asking the same questions that I had so I decided to make the guide I never got. It goes beyond getting started and explains customization/security. You can read it here: [https://medium.com/@christopher.mathews/how-to-self-host-supabase-on-your-own-computer-full-guide-cceac9b740ea](https://medium.com/@christopher.mathews/how-to-self-host-supabase-on-your-own-computer-full-guide-cceac9b740ea )

**Comment by timothybennette8** [comment]
Thank you for your time and effort - this looks to be just what I needed!
